### PR TITLE
Mh/19 banner image link to home

### PR DESCRIPTION
**Pull request by heaversm**
add home links to banner and sidenav logo

Fixes  https://trello.com/c/w5x1Pm5S/19-banner-image-link-to-home

### CODE REVIEW

**Comment by skyfallsin** [comment]
@heaversm we lost the id tags in the html content.

File: templates/ai/content/ai-basics/index-content.html [hunk 1]
```diff
@@ -1,16 +1,16 @@
-<h2 id="the-quick--dirty-intro-to-ai">The quick &amp; dirty intro to AI</h2>
-<h4 id="what-is-ai-in-2023">What is AI in 2023?</h4>
```
Comment (skyfallsin) [inline templates/ai/content/ai-basics/index-content.html:2]: these ids are needed for sidebar scroll

File: README.md [hunk 2]
```diff
@@ -38,6 +38,8 @@ npm run build
 
 The Webpack will output all HTML files to a folder called `dist`.
 
+Turn markdown to html: `tools/build_ai_guide.sh`
```
Comment (skyfallsin) [inline README.md]: this doesn't apply to the main future website, we should likely call this out in a separate README

File: templates/ai/content/readme/index.md [hunk 3]
```diff
@@ -13,3 +13,5 @@ Mozilla has been developing and shipping AI through its products and initiatives
 But like many others, we also see the risk of real-world harm. We don’t have to cast our gazes to some far-flung science fiction future to envision it: people are already experiencing real-world impacts from this technology, right here and now.
 
 Much will depend on how this technology is developed and deployed, but it will also depend on who does it. We believe that the open source community has a critical role to play in ensuring that AI develops in a way that is responsible, trustworthy, equitable, and safe. We hope that this guide contributes to mobilizing the open source community to join us in this important work.
+
+[Next Section: AI Basics](/ai/content/ai-basics/)
```
Comment (skyfallsin) [inline templates/ai/content/readme/index.md]: we can use HTML tags in the MD files, and we already have some styling for this button in index.html, please updated

**Review by skyfallsin — CHANGES_REQUESTED** [review]
the templates/ai/content *.html files should only be updated using the tools script

File: templates/ai/content/readme/index.md (hunk 3 above)
Comment (heaversm) [reply templates/ai/content/readme/index.md]: Don't know what you're asking for here, sorry 🔼. I also don't see any buttons on index.html

File: templates/ai/content/ai-basics/index-content.html (hunk 1 above)
Comment (heaversm) [reply templates/ai/content/ai-basics/index-content.html:2]: @skyfallsin  - I don't know how to fix this with your build script. Can you provide the markdown example? How did these all get overwritten?

Comment (heaversm) [reply templates/ai/content/ai-basics/index-content.html:2]: @skyfallsin The build script is not generating the ids properly for me.

**Pull request merged**